### DISCUSSION TITLE
Why format in config file doesnt work in recursive mode?

### DISCUSSION THREAD

**Discussion by pcastro-handover** [terraform-linters/tflint#2167] [Q&A] [answered]
Hey, sorry if this was asked or documented somewhere else (couldn't find it), but is there any particular reason for the bellow?

> In recursive mode (--recursive), this field will be ignored in configuration files and must be set via a flag.

**Comment by bendrucker** [comment, marked as answer]
Configuration options can change per module recursed into. Format cannot, as it's used at the end to print the result.

**Reply by pcastro-handover** [reply]
got it. thanks!!
